src/views/SelectJobsModal.vue

```diff
@@ -92,7 +96,9 @@ export default defineComponent({
   methods: {
     async search(event: any) {
       this.queryString = event.target.value.trim();
-      if(this.queryString.length > 0) this.getJobs();
+      if(this.queryString.length > 0) {
```

Comment:
Why do we need the length check at all?

_"That is, all values are truthy except false, 0, -0, 0n, "", null, undefined, and NaN."_

https://developer.mozilla.org/en-US/docs/Glossary/Truthy

Reply:
Yes, sir. The empty queryString itself is a false value.